docs/scripts/script.min.js

```diff
@@ -29480,29 +29489,29 @@ SpecialEvents.events = [];
 // TODO: Fetch events from a server each 1/2/3/6/12/24 hours?
 // Create our events here for now
 // Yearly
-SpecialEvents.newEvent('Flying Pikachu', 'Encounter Flying Pikachu for a limited time roaming Kanto.', 
```

Comment:
I think Atom on Windows is proactively trying to change some file endings. Also, I think git hates me

Reply:
The `docs` file shouldn't be update, we only update this when we push to master for the live website.

Reply:
Understood! Thanks for (another) assist :)